R/Simulations-class.R

```diff
@@ -179,32 +179,36 @@ Simulations <- function(fit,
   )
 }
 
-# nolint start
-##' Class for the simulations output from dual-endpoint model based designs
-##'
-##' This class captures the trial simulations from dual-endpoint model based
-##' designs. In comparison to the parent class \code{\linkS4class{Simulations}},
-##' it contains additional slots to capture the dose-biomarker fits, and the
-##' sigma2W and rho estimates.
-##'
-##' @slot rhoEst the vector of final posterior median rho estimates
-##' @slot sigma2West the vector of final posterior median sigma2W estimates
-##' @slot fitBiomarker list with the final dose-biomarker curve fits
-##'
-##' @export
-##' @keywords classes
+# DualSimulations ----
+
+## class ----
+
+#' `DualSimulations`
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' This class captures the trial simulations from dual-endpoint model based
+#' designs. In comparison to the parent class [`Simulations`],
+#' it contains additional slots to capture the dose-biomarker fits, and the
+#' sigma2W and rho estimates.
+#'
+#' @slot rho_est (`numeric`)\cr vector of final posterior median rho estimates
+#' @slot sigma2_west (`numeric`)\cr vector of final posterior median sigma2W estimates
+#' @slot fit_biomarker (`list`)\cr with the final dose-biomarker curve fits
+#' @aliases DualSimulations
+#' @export
 .DualSimulations <-
   setClass(
     Class = "DualSimulations",
-    representation(
-      rhoEst = "numeric",
-      sigma2West = "numeric",
-      fitBiomarker = "list"
+    slots = c(
+      rho_est = "numeric",
+      sigma2_west = "numeric",
```

Comment:
```suggestion
      sigma2w_est = "numeric",
```
And similar elsewhere